plugin.yaml

```diff
@@ -2,128 +2,16 @@ version: 0.1
 required_trunk_version: ">0.17.0"
 actions:
   definitions:
-    - id: commitlint
-      runtime: node
-      packages_file: actions/commitlint/package.json
-      run: commitlint --edit ${1}
-      triggers:
-        - git_hooks: [commit-msg]
-
     - id: git-lfs
```

Comment:
Is there a reason this and `buf-gen` are still in the plugin root? Since these are both actions, why not put this in actions/ or have a misc folder in there?

Reply:
Good call.